Enable/Views/MapView.m

```diff
@@ -0,0 +1,88 @@
+//
+//  MapView.m
+//  Enable
+//
+//  Created by Amanda Wang on 7/5/22.
+//
+
+#import "MapView.h"
+#import <CoreLocation/CoreLocation.h>
+@implementation MapView{
+    BOOL firstLocationUpdate;
+}
+
+/*
+// Only override drawRect: if you perform custom drawing.
+// An empty implementation adversely affects performance during animation.
+- (void)drawRect:(CGRect)rect {
+    // Drawing code
+}
+*/
+
+- (instancetype) initWithCoder:(NSCoder *)aDecoder{
+    self = [super initWithCoder:aDecoder];
+    if (self){
+        [self mapInit];
+    }
+    return self;
+}
+- (instancetype) initWithFrame:(CGRect)frame{
+    self = [super initWithFrame:frame];
+    if (self){
+        [self mapInit];
+    }
+    return self;
+}
+- (instancetype) mapInit{
+    [[NSBundle mainBundle] loadNibNamed: @"MapView" owner: self options:nil];
+    [self addSubview: self.contentView];
+    self.contentView.frame = self.bounds;
+    firstLocationUpdate = NO;
+    //default camera value, Sydney, Australia
```

Comment:
it does default to the user location, however i don't want a null camera value so I default it to this in case user does not provide location permission.